advanced/src/components/Post.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export default class Post extends React.Component {
+  render() {
+    let title = this.props.post.title
+    if (this.props.isDraft) {
+      title = `${title} (Draft)`
+    }
+
+    return (
```

Comment:
An anchor can't be a child of another anchor. (`<Link /> maps to an anchor tag in the DOM`) Try this:

```js
<Link
  className="no-underline ma1 db pv4 ph3 ph0-l no-underline black dim"
  to={`/post/${this.props.post.id}`}
>
  <article className="bb b--black-10">
    <div className="flex flex-column flex-row-ns">
      <div className="w-100 w-60-ns pl3-ns">
        <h1 className="f3 fw1 baskerville mt0 lh-title">{ title }</h1>
        <p className="f6 f5-l lh-copy">
          {this.props.post.text}
        </p>
        <p className="f6 lh-copy mv0">By { this.props.post.author.name }</p>
      </div>
    </div>
  </article>
</Link>
```

Reply:
Whoops. Didn't get that one right. Anyway, the code is just below `return`

Reply:
Done